console/console-init/ui/src/utils/utils.ts

```diff
@@ -256,6 +256,10 @@ const getJson = (objects: any[]) => {
   return object;
 };
 
+const createDeepCopy = (object: any) => {
```

Comment:
Again, please rename object to something descriptive.

Reply:
I think this is a generic function and name is descriptive as per the function